furusiya/model/entities/party/player.py

```diff
@@ -48,6 +48,15 @@ def unmount(self, horse):
             self.mounted = False
             horse.is_mounted = False
 
+    def rest(self):
+        if config.data.features.allowResting:
```

Comment:
This should be moved beside the `elif`, eg. `elif user.input == '.' and config.data.features.allowResting`